doc/whats_new/v1.1.rst

```diff
@@ -28,12 +28,29 @@ parameters, may produce different models from the previous version. This often
 occurs due to changes in the modelling logic (bug fixes or enhancements), or in
 random sampling procedures.
 
+:mod:`sklearn.cluster`
+......................
+
 - |Efficiency| :class:`cluster.KMeans` now defaults to ``algorithm="lloyd"``
   instead of ``algorithm="auto"``, which was equivalent to
   ``algorithm="elkan"``. Lloyd's algorithm and Elkan's algorithm converge to the
   same solution, up to numerical rounding errors, but in general Lloyd's
   algorithm uses much less memory, and it is often faster.
 
+- |Fix| :func:`manifold.spectral_embedding` now uses Gaussian instead of
+  the previous uniform on [0, 1] random initial approximations to eigenvectors
+  in eigen_solvers `lobpcg` and `amg` to improve their numerical stability,
+  which might produce different clusters in spectral clustering.
+  :pr:`21565` by :user:`Andrew Knyazev <lobpcg>`.
```

Comment:
We only need to add a single entry in the "Changed models" section. Something like this:

```rst
- |Fix| The eigenvectors initialization for :class:`cluster.SpectralClustering`
  and :class:`manifold.SpectralEmbedding` now samples from a Gaussian when
  using the `'amg'` or `'lobpcg'` solver. This change  improves numerical
  stability, but may result in a different model.
```

Looking at the other "Change Model" sections, we normally do not include the pr number or author in the "Change Model" section. That information can be obtained in the Changelog below, which you already include in this PR.